module/behaviour/strategy/SoccerStrategy/src/SoccerStrategy.cpp

```diff
@@ -350,71 +377,85 @@ namespace module::behaviour::strategy {
     }
 
     void SoccerStrategy::normalReady(const GameState& gameState, const FieldDescription& fieldDescription) {
-        if (gameState.data.our_kick_off) {
-            walkTo(fieldDescription, cfg_.start_position_offensive);
+        log<NUClear::DEBUG>("normalReady()");
+
+        // auto walkTarget = gameState.data.our_kick_off ? cfg_.start_position_offensive : cfg_.start_position_defensive;
+        // walkTo(fieldDescription, walkTarget);
+
+        if (!startedWalkingToReady) {
+            startedWalkingToReadyAt = NUClear::clock::now();
+            startedWalkingToReady = true;
         }
-        else {
-            walkTo(fieldDescription, cfg_.start_position_defensive);
+
+        if (NUClear::clock::now() - startedWalkingToReadyAt < std::chrono::milliseconds(17*1000)) {
+            emit(std::make_unique<MotionCommand>(utility::behaviour::WalkToReady()));
+        } else {
+            log("10 seconds up, stopping walk");
+            standStill();
         }
-        find({FieldTarget(FieldTarget::Target::SELF)});
+
+        // Self localise while we're walking on in READY
+        // find({FieldTarget(FieldTarget::Target::SELF)});
+
         currentState = Behaviour::State::READY;
     }
 
     void SoccerStrategy::normalSet() {
+        log<NUClear::DEBUG>("normalSet()");
         standStill();
-        find({FieldTarget(FieldTarget::Target::BALL)});
+        // find({FieldTarget(FieldTarget::Target::BALL)});
```

Comment:
If you're in normal set, shouldn't you be finding yourself? - someone cool, probably